sdk/src/types/block/error.rs

```diff
@@ -245,6 +249,10 @@ impl fmt::Display for Error {
             }
             Self::InvalidNativeTokenCount(count) => write!(f, "invalid native token count: {count}"),
             Self::InvalidNetworkName(err) => write!(f, "invalid network name: {err}"),
+            Self::InvalidEpochNearingThreshold(err) => write!(f, "invalid epoch nearing threshold: {err}"),
+            Self::InvalidManaDecayFactors(err) => write!(f, "invalid mana decay factors: {err}"),
+            Self::InvalidLivenessThreshold(err) => write!(f, "invalid liveness threshold: {err}"),
+            Self::InvalidOption(err) => write!(f, "invalid work score structure: {err}"),
```

Comment:
It's not limited to work score structure?

Reply:
oh whoops